src/Nethermind/Nethermind.Trie/VerkleTree.cs

```diff
@@ -220,15 +244,15 @@ public byte[] GetTreeKeyForStorageSlot(Address address, UInt256 storageKey)
         
     public void SetCode(Address address, byte[] code)
     {
-        byte[][] chunkifiedCode = chunkifyCode(code);
+        byte[][] chunkifiedCode = ChunkifyCode(code);
         byte[] chunkKey;
         for (int i = 0; i < chunkifiedCode.Length; i++)
         {
             chunkKey = GetTreeKeyForCodeChunk(address, (UInt256)i);
```

Comment:
Does key for chunk need Uint256 arithmetic? (maybe yes)

Reply:
yes, that's necessary